pipeline/backend/kubernetes/kubernetes.go

```diff
@@ -45,7 +44,10 @@ const (
 	EngineName = "kubernetes"
 )
 
-var noContext = context.Background()
+var (
+	noContext            = context.Background()
```

Comment:
I'd like to get rid of **noContext** !!!

Reply:
Agree. Check if did it right.